news/_posts/2021-03-02-archived-modules.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "Focusing on essential modules"
+subtitle: "Announcing end of official support for less popular modules"
+date:   2021-03-02 14:00:00
+authors:  [ MarionFrancois ]
+icon: icon-puzzle
+tags: 
+ - modules
+ - news
+---
+
+The maintainers team resources are limited, and there are too many free and open source modules on the PrestaShop GitHub organization to be able to maintain them all. Because of this maintenance burden, some of these modules have not been given a new release for years.
+
+For this reason, the product team recently did a survey of almost 600 PrestaShop users to find out which modules were essential for their business and which ones they used the least.
+
+Following this survey, the product team and the developer team decided to maintain a smaller set of modules that are really important for the users, rather than attempting to keep all modules afloat which results in a poor maintenance of each and a constant struggle to update everything.
```

Comment:
```suggestion
As a result of this survey, the teams have decided to focus their efforts where it really matters. Rather than keeping hundreds of sparsely updated modules, starting this year, the PrestaShop Project will only commit to maintaining a smaller set of them – the ones that are really important for users.
```